Aptoide/aptoide-client-v8
# [bugfix/MOB-134] Fixed crashes for API < 21 (& home chips)

**What does this PR do?**

   This PR fixes several reasons for crashes in devices with API < 21 because of dark theme development. It also fixes visual problems with home chips.

**Database changed?**

   No

**How should this be manually tested?**

  Ideally, try to test every screen and check that the app does not crash or behave strangely in devices with API < 21. Also check if home chips are now visually ok when you select and deselect them.

**What are the relevant tickets?**

  [MOB-134](https://aptoide.atlassian.net/browse/MOB-134)

**Code Review Checklist**
